flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateTable.java

```diff
@@ -236,12 +232,12 @@ public String getColumnSqlString() {
 		writer.startList("", "");
 		for (SqlNode column : columnList) {
 			writer.sep(",");
-			if (column instanceof SqlTableColumn) {
-				SqlTableColumn tableColumn = (SqlTableColumn) column;
-				tableColumn.getName().unparse(writer, 0, 0);
-			} else {
-				column.unparse(writer, 0, 0);
+			SqlTableColumn tableColumn = (SqlTableColumn) column;
+			if (tableColumn.isComputed()) {
+				tableColumn.getExpr().unparse(writer, 0, 0);
```

Comment:
We can encapsulate all the unparse logic in `SqlTableColumn`. E.G. the code can be as simple as:
```java
column.unparse(writer, 0, 0)
```